etcdutl/etcdutl/hashkv_command.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2024 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package etcdutl
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+
+	"go.etcd.io/etcd/pkg/v3/cobrautl"
+	"go.etcd.io/etcd/server/v3/storage/backend"
+	"go.etcd.io/etcd/server/v3/storage/mvcc"
+)
+
+var (
+	hashKVRevision int64
+)
+
+// NewHashKVCommand returns the cobra command for "hashkv".
+func NewHashKVCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "hashkv <filename>",
+		Short: "Prints the KV history hash of a given file",
+		Run:   hashKVCommandFunc,
+	}
+	cmd.Flags().Int64Var(&hashKVRevision, "rev", 0, "maximum revision to hash (default: latest revision)")
+	return cmd
+}
+
+func hashKVCommandFunc(cmd *cobra.Command, args []string) {
+	if len(args) != 1 {
+		err := fmt.Errorf("hashkv requires exactly one argument")
+		cobrautl.ExitWithError(cobrautl.ExitBadArgs, err)
```

Comment:
Suggest to use `cobra.ExactArgs(1)`, but not a blocker to merge this PR.

Refer to https://github.com/etcd-io/bbolt/blob/f663d4e6043e37fa77c3b554f2692d4d02cc8c95/cmd/bbolt/command_surgery_freelist.go#L31

Reply:
Updated.